k-distribution/tests/profiling/setup_profiling.sh

```diff
@@ -6,8 +6,11 @@ export DEBIAN_FRONTEND=noninteractive
 export BENCHER_VERSION="0.3.10"
 apt-get update
 apt-get upgrade --yes
```

Comment:
Missed this last time but you may want to be warry of blind upgrades, they could have unintended changes for tools / builds. Just a note.

Reply:
Should I remove them?